src/txmempool.h

```diff
@@ -92,17 +92,23 @@ class CTxMemPoolEntry
     size_t nTxWeight;          //!< ... and avoid recomputing tx weight (also used for GetTxSize())
     size_t nModSize;           //!< ... and modified size for priority
     size_t nUsageSize;         //!< ... and total memory usage
+
 public:
     mutable size_t vTxHashesIdx; //!< Index in mempool's vTxHashes
+
 private:
     unsigned int entryHeight;  //!< Chain height when entering the mempool
 
     // Space that is typically used as padding. Commented out so that it doesn't cause extra memory use on different architectures.
     // char PADDING_FREE_SPACE[2];
 
     // Bool flags
-    bool hadNoDependencies;    //!< Not dependent on any other txs when it entered the mempool
-    bool spendsCoinbase;       //!< keep track of transactions that spend a coinbase
+    struct CTxMemPoolEntryFlags {
+        //int UNUSED_BITS : 7;
+        bool spendsCoinbase : 1;       //!< keep track of transactions that spend a coinbase
```

Comment:
Kind of a lonely bitfield now with just one bool. But maybe it will have some more flags later.